awx/main/access.py

```diff
@@ -38,6 +38,7 @@
     Group,
     Host,
     Instance,
+    InstanceLink,
```

Comment:
This change is no longer necessary, and will probably trigger a linting fail in CI.